Source/EasyNetQ/Persistent/ConnectionExtensions.cs

```diff
@@ -0,0 +1,15 @@
+using RabbitMQ.Client;
+using RabbitMQ.Client.Exceptions;
+
+namespace EasyNetQ.Persistent;
+
+internal static class ConnectionExtensions
+{
+    public static void EnsureIsOpen(this IConnection connection)
```

Comment:
I think `AssertIsOpen` will be better than `Ensure` because it throws exception in case if closed and do not try to open